lib/application_group.ts

```diff
@@ -0,0 +1,224 @@
+import { RawStorageKey } from '@Lib/storage_keys';
+import { removeFromArray, findInArray } from '@Lib/utils';
+import { UuidString, DeinitSource } from './types';
+import { SNApplication } from './application';
+import { PureService } from '@Services/pure_service';
+import { DeviceInterface } from '@Lib/device_interface';
+import { Uuid } from '@Lib/uuid';
+
+export type ApplicationDescriptor = {
+  identifier: string | UuidString,
```

Comment:
Why not just a `UuidString`?

Reply:
I believe `string` is for the default identifier: `standardnotes` and `UuidString` is for the additional app descriptors.